Slight mention of local signers and clef in the worklock CLI guide (#2217)
nucypher/nucypher: docs/source/guides/worklock_guide.rst

        Pending amount of work required before all of the participant's escrowed ETH will be refunded

.. note::

>     --signer is not required if you are running a local ethereum node or your --provider is the

Reviewer: ```suggestion
    ``--signer`` is not required if you are running a local ethereum node or your ``--provider`` is the
```

>     same entity as your singer.   If you are using a remote hosted etheruem node, you will need to run a local

Reviewer: ```suggestion
    same entity as your signer.   If you are using a remote hosted ethereum node, you will need to run a local
```

>     signer like clef.

Reviewer 2: ```suggestion
    signer like clef. See :ref:`using-eth-node`.
```
